packages/expo-cli/src/commands/publish.ts

```diff
@@ -46,6 +57,9 @@ export async function action(
   }
   log(`- Release channel: ${log.chalk.bold(options.releaseChannel)}`);
   log(`- Workflow: ${log.chalk.bold(target.replace(/\b\w/g, l => l.toUpperCase()))}`);
+  if (user.kind === 'robot') {
```

Comment:
As mentioned, we could also either show this "all the time" or `if (user.username !== owner)`. That means it's always visible for robot users, but also visible for normal users whenever `owner` is defined in the manifest.

Reply:
alternatively, we could also render something like `App: ...` or `Project: ...` with `@<owner>/<name>`

Reply:
I like the second suggestion, with `Project: @<owner>/<name>`. This should perhaps be the first bullet point?